doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Uno.GetStarted.Licensing
+---
+
+# Sign in with Uno Platform
+
+Sign in with your Uno Platform account directly in your favorite IDE—Visual Studio, VS Code, or Rider—to unlock powerful tools like Hot Reload, helping you speed up development. With a single registration, you also get early access to new features and the opportunity to connect with the Uno community to share feedback and network.
```

Comment:
```suggestion
Sign in with your Uno Platform account directly in your favorite IDE —Visual Studio, VS Code, or Rider— to unlock powerful tools like Hot Reload, helping you speed up development. With a single registration, you also get early access to new features and the opportunity to connect with the Uno community to share feedback and network.
```
@jeromelaban, @sasakrsmanovic, @kazo0 should we mention "powerful tools such as the Hot Reload indicator" instead here?

Reply:
Also not sure about this sentence @sasakrsmanovic:
"you also get early access to new features and the opportunity to connect with the Uno community to share feedback and network."

Reply:
It is just missing Platform in Uno. Other than that I don't see an issue